.github/workflows/build-images-ci.yaml

```diff
@@ -54,6 +54,10 @@ jobs:
             dockerfile: ./images/operator/Dockerfile
             platforms: linux/amd64,linux/arm64
 
+          - name: hubble
+            dockerfile: ./images/hubble/Dockerfile
+            platforms: linux/amd64,linux/arm64
+
```

Comment:
wondering if it's worth pushing CI images for Hubble CLI instead of only pushing release images 💭 

also not sure if it's worth adding hubble cli image to beta and hotfix builds 💭 

- https://github.com/cilium/cilium/blob/main/.github/workflows/build-images-beta.yaml
- https://github.com/cilium/cilium/blob/main/.github/workflows/build-images-hotfixes.yaml

@cilium/hubble-maintainers any preferences?

Reply:
While I think we should build them for sure, I don't see much value in pushing Hubble CLI CI images. 

Does Cilium CLI have Docker images? What's our policy there?

Reply:
not sure if we have any policy defined. right now cilium-cli only push CI images, but there is some interest for getting release images published 🧠 

- https://github.com/cilium/cilium-cli/issues/2780
- https://github.com/cilium/cilium-cli/issues/2779

Reply:
food for thought https://github.com/cilium/hubble/pull/1631

Reply:
looks like we are going with https://github.com/cilium/hubble/pull/1631. closing this PR 🚀